app/src/main/java/no/usn/mob3000/ui/screens/auth/CreateUserScreen.kt

```diff
@@ -85,7 +85,7 @@ fun CreateUserScreen(
                 Spacer(modifier = Modifier.height(16.dp))
 
                 Button(
-                    onClick = { onSignUpClick() },
+                    onClick = { onSignUpClick(mail, password) },
```

Comment:
Single-use functions do not need brackets. The brackets were only there before in case we needed to do some finagling with additional functions. So this should be `onClick = onSignUpClick(mail, password)`.

Reply:
Also, kind of forgot it in the moment, but you're not doing a password check here. We have two fields and `onSignUpClick` should only be called if both are equal values. If they're not, an error message should display, telling the end-user that both passwords need to be identical.

Reply:
> Single-use functions do not need brackets. The brackets were only there before in case we needed to do some finagling with additional functions. So this should be `onClick = onSignUpClick(mail, password)`.

I know it's not necessary, but I'm using it as a default practice, because it's more practical if changes are applied later on in the future.